docs/INSTALLATION.md

```diff
@@ -0,0 +1,233 @@
+* `s3Client__ServiceURL` - The service URL of the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `ServiceHostUrl` - Hostname (or IP address + port) of the corresponding Asset Management Service.
+* `MariaDB__ConnectionString` - The connection string used by the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager) to connect to [MariaDB](https://mariadb.org/) database. The format of the connection string must always be similar to what is provided, but the port number, database name, username and password must change accordingly if their default values were changed.
+* `AssetManagerHostUrl` - Hostname (or IP address + port) of the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager).
+
+### Running the Docker Compose builds
+
+OVE provides separate installers to help users install the necessary components. To install the core components of OVE run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.ove.yml create
+```
+
+To install the [**OVE Asset Services**](https://github.com/ove/ove-asset-services) run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.assets.yml create
+```
+
+Once the installation procedure has completed, the successful installation of OVE can be verified by accessing the [OVE Core API Documentation](http://OVE_CORE_HOST:PORT/api-docs) on a web browser.
+
+### Starting and stopping OVE
+
+To start the Docker runtime run:
+
+```sh
+docker-compose -f docker-compose.assets.yml up -d
+```
+
+To stop the Docker runtime run:
+
+```sh
+docker-compose -f docker-compose.assets.yml down
+```
+
+To clean-up the Docker runtime first stop any active instances and then run:
+
+```sh
+docker system prune
+docker volume prune
+```
+
+## Installation by downloading and compiling source code
+
+All OVE projects use a build system based on [Lerna](https://lernajs.io/). [**OVE Asset Services**](https://github.com/ove/ove-asset-services) are based on [.NET Core](https://github.com/dotnet/core). All other OVE projects are based on [Node.js](https://nodejs.org/en/) and are compiled with [Babel](http://babeljs.io/) and deployed on a [PM2](http://pm2.keymetrics.io/) runtime.
+
+### Prerequisites
+
+* [git](https://git-scm.com/downloads)
+* [Node.js](https://nodejs.org/en/) (v8.0+)
+* [NPM](https://www.npmjs.com/)
+* [NPX](https://www.npmjs.com/package/npx): `npm install -global npx`
+* [PM2](http://pm2.keymetrics.io/): `npm install -global pm2`
+* [Lerna](https://lernajs.io/):  `npm install -global lerna`
+
+Compiling source code for the Docker runtime also require:
```

Comment:
I would say that it *only* requires Docker and Docker Compose, as OVE's dependencies are automatically downloaded and installed inside the Docker container when the Dockerfile is run.

Reply:
Build script builds locally first. Also, some people may want to build locally not using the build script and then build it for Docker as well. This is highly subjective and can be very confusing to explain the scenarios separately. It would be better for them to install all the tools they require to develop locally, since this is the intended audience for this type of installation.